sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -101,12 +104,66 @@ def get_token(self, *args):
 
 
 class ContainerRegistryTestClass(AzureTestCase):
-
     def __init__(self, method_name):
         super(ContainerRegistryTestClass, self).__init__(method_name)
         self.vcr.match_on = ["path", "method", "query"]
         self.recording_processors.append(AcrBodyReplacer())
 
+    def sleep(self, t):
+        if self.is_live:
+            time.sleep(t)
+
+    def _fake_sleep(self, *args, **kwargs):
```

Comment:
is this used anywhere or do you have future plans for it?

Reply:
Nope, I had plans to use it but I'm going to remove it for now.